openmeetings-web/src/main/java/org/apache/openmeetings/web/app/Application_de.properties.xml

```diff
@@ -981,4 +981,7 @@ Bitte am OM server {3} anmelden und sie unter Admininistration -> Gruppen prüfe
 	<entry key="wizard.button.finish"><![CDATA[Ende]]></entry>
 	<entry key="zoom.FULL_FIT"><![CDATA[Vollformat]]></entry>
 	<entry key="zoom.PAGE_WIDTH"><![CDATA[Seitenbreite]]></entry>
+	<entry key="wb.add.whiteboard"><![CDATA[Add new whiteboard to conference room]]></entry>
```

Comment:
German is missing :(

Reply:
You mean its not translated ? Cause the strings are there.

Reply:
yes, not translated :((